core/server/master/src/main/java/alluxio/master/file/options/CreatePathOptions.java

```diff
@@ -169,6 +183,17 @@ public T setMode(Mode mode) {
     return getThis();
   }
 
+  /**
+   * Sets an immutable copy of acl as the internal access control list.
+   *
+   * @param acl the ACL entries
+   * @return the updated options object
+   */
+  public T setAcl(List<AclEntry> acl) {
+    mAcl = ImmutableList.copyOf(acl);
```

Comment:
if on the `set` the acl is immutable, then can we just return it for `getAcl`?

Reply:
done